thoth/report_processing/components/adviser.py

```diff
@@ -142,170 +138,235 @@ def _aggregate_thoth_results_from_ceph(
+                            if info["type"] == "ERROR":
+                                justifications_collected.append(
+                                    {
+                                        "document_id": document_id,
+                                        "date": datetime_object,
+                                        "analyzer_version": analyzer_version,
+                                        "justification": {"message": info["message"], "type": info["type"]},
+                                        "error": True,
+                                        "message": info["link"],
+                                        "type": info["type"],
+                                    }
+                                )
+                            else:
+                                justifications_collected.append(
+                                    {
+                                        "document_id": document_id,
+                                        "date": datetime_object,
+                                        "analyzer_version": analyzer_version,
+                                        "justification": {"message": info["message"], "type": info["type"]},
+                                        "error": False,
+                                        "message": info["link"],
+                                        "type": info["type"],
+                                    }
+                                )
+                        else:
+                            if info["type"] == "ERROR":
+                                justifications_collected.append(
+                                    {
+                                        "document_id": document_id,
+                                        "date": datetime_object,
+                                        "analyzer_version": analyzer_version,
+                                        "justification": {"message": info["message"], "type": info["type"]},
+                                        "error": True,
+                                        "message": info["message"],
+                                        "type": info["type"],
+                                    }
+                                )
+                            else:
+                                justifications_collected.append(
+                                    {
+                                        "document_id": document_id,
+                                        "date": datetime_object,
+                                        "analyzer_version": analyzer_version,
+                                        "justification": {"message": info["message"], "type": info["type"]},
+                                        "error": False,
+                                        "message": info["message"],
+                                        "type": info["type"],
+                                    }
+                                )
```

Comment:
seems that this is the same as this: https://github.com/thoth-station/report-processing/pull/86/files#r553006092